PowerShellTools/Classification/PowerShellClassifier.cs

```diff
@@ -141,6 +135,35 @@ private void FillClassificationGap(List<ClassificationSpan> classifications, Spa
             }
         }
 
+        private static void CreateClassificationTypeMap()
+        {
+            if (_tokenClassificationTypeMap == null || _tokenClassificationTypeMap.Count == 0)

```

Comment:
Nit, stylistic but /technically/ better [can fail early] !_tokenClassificationTypeMap.Any() instead of .Count ==0